### ISSUE TITLE
utf8 file with BOM

### ISSUE THREAD

**Issue by karulis** [behave/behave#361]
Visual Studio by default saves files as utf8 with BOM,
and that causes error:

```python
...
  File "C:\Python34\lib\site-packages\behave\runner.py", line 303, in exec_file
    code = compile(f.read(), filename2, 'exec')
  File "steps\plot.py", line 1
    ď»ż"""Data analizing and plotting module"""
      ^
SyntaxError: invalid character in identifier
```

**Comment by karulis** [comment]
ok so it can be fixed by:
```javascript
def exec_file(filename, globals={}, locals=None):
    if locals is None:
        locals = globals
    locals['__file__'] = filename
    with open(filename, "rb") as f:
        # -- FIX issue #80: exec(f.read(), globals, locals)
        # try:
        filename2 = os.path.relpath(filename, os.getcwd())
        code = compile(f.read(), filename2, 'exec')
        exec(code, globals, locals)
        # except Exception as e:
        #     e_text = _text(e)
        #     print("Exception %s: %s" % (e.__class__.__name__, e_text))
        #     raise
```

**Comment by jenisys** [comment]
INTEGRATED INTO: behave-1.2.6.dev0

But note that pypy does not support python files with BOM (=> SyntaxError).
